docs/user/02-10-usage.md

```diff
@@ -97,3 +100,37 @@ After successfully installing your Secret, you can create a ServiceInstance and
     ```bash
     kubectl delete -f hack/operator-secret.yaml
     ```
+
+## Create Service Instance with custom secret
+
+When you create Service Instance you can use field `btpAccessCredentialsSecret` in spec of Service Instance, you can pass there secret from kyma-system and it will be used to create instance. You can use different secret for different Service Instances.
+Once you set Secret name on ServiceInstance you cannot change it future.
+Linking BTP Credentials with Service Instance will set in status also subaccount ID to which  one credentials belong.
```

Comment:
Adding the access credentials of the SAP BTP Service Manager Instance in your ServiceInstance results in displaying the subaccount ID to which the instance belongs in the **subaccount ID** field.